### PR TITLE
fix: logger plugins default format not works

### PR DESCRIPTION
**Pull request by Vacant2333**
### Description

- make the default log format works
- remove the logger's unused codes

Part of https://github.com/apache/apisix/issues/10494

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: apisix/utils/log-util.lua [hunk 1]
```diff
@@ -152,53 +202,14 @@ local function get_full_log(ngx, conf)
         service_id = service_id,
         route_id = route_id,
         consumer = consumer,
-        client_ip = core.request.get_remote_client_ip(ngx.ctx.api_ctx),
+        client_ip = core.request.get_remote_client_ip(ctx),
```
[the remark this answers is not in the document]
Comment (Vacant2333) [reply apisix/utils/log-util.lua:205]: https://github.com/apache/apisix/pull/10599/files/3b5ae9cb9e6e32e1725a973a4b13a95939ec92e4#diff-34cfcd7ea6de7a49aca1445c26a8f93b0e10bccb2d690ffbaf098b6678ab0b55R159
its the same

File: .gitignore [hunk 2]
```diff
@@ -84,6 +84,7 @@ t/plugin/grpc-web/grpc-web-server
 t/plugin/grpc-web/node_modules/
 pack
 pack-v*-linux.tgz*
+etcd_data/
```
Comment (moonming) [inline .gitignore:87]: Please ensure that you do not make any unnecessary changes or modifications that are not relevant to the task

**Review by moonming — CHANGES_REQUESTED** [review]
Can you tell me the cause of the bug? How did you fix it? I don’t understand this PR at all
